### ISSUE TITLE
Userland: Icons are being looked for in the wrong place

### ISSUE THREAD

**Issue by Alizter** [SerenityOS/serenity#20889]
When opening the man pages for an application, for instance `Demos/Eyes`, the man pages appear to point to the wrong place. I get the following error:
![image](https://github.com/SerenityOS/serenity/assets/8614547/a20c03db-e2da-41ee-aa0a-de27296f2b04)
I've checked and the man page is trying to point to the path:
```
/res/icons/16x16/app-eyes.png
```
This file does exist. However it seems that the man page viewer (perhaps) is not taking this as an absolute path but rather turning it into a relative one; thus looking for it in the wrong place:
```
/usr/share/man/man1/Applications/res/...
```
I'm not sure how the paths in the man pages are resolved but I suspect that it is not handling absolute paths correctly.

I believe that this is a general issue with the `Help` application.

I am on revision `24bc761`.

**Comment by Wissotsky** [comment]
Can concur the same exact issue happens on rev `50bb785`

**Comment by kemzeb** [comment]
This is also mentioned in #20041, with a PR that hasn't seen any recent activity in a while.

**Comment by Alizter** [comment]
Fix in #20902.
